packages/workflow-compile/index.js

```diff
@@ -14,6 +14,8 @@ const SUPPORTED_COMPILERS = {
   external: require("@truffle/external-compile").Compile
 };
 
+const { TruffleDB } = require("@truffle/db");
```

Comment:
Did this get added to workflow-compile/package.json? We'll need to make sure it's there

(And, noticing the webpack.config.js change, we'll need to add it as an optionalDependency to truffle, if it's not already there)